src/model/calculations.ts

```diff
@@ -100,11 +101,154 @@ export class OptionsRemainingCalculator {
   }
 }
 
+interface StockClass {
+  object_type: string;
+  id: string;
+  name: string;
+  class_type: string;
+  default_id_prefix?: string;
+  current_shares_authorized?: string;
```

Comment:
**Non-blocking:** Generally speaking, let's avoid including attributes that are not used.